# LuxOS: zdroj z gitu -> vydany archiv tiff-3.9.7.tar.gz z download.osgeo.org (oficialni vydani libtiff)
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,16 +14,15 @@
   lib32-libjpeg-turbo
   lib32-zlib
 )
-makedepends=(git)
 source=(
-  git+https://gitlab.com/libtiff/libtiff.git#tag=v${pkgver}
+  https://download.osgeo.org/libtiff/tiff-${pkgver}.tar.gz
   libtiff4-soname.patch
 )
-sha256sums=('b4fe4a431427dcabeac32374419271fc249453748c7e21b73c63928e668c1e54'
+sha256sums=('f5d64dd4ce61c55f5e9f6dc3920fbe5a41e02c2e607da7117a35eb5c320cef6a'
             'e6246631f6d943715b612b4ddd2279c946ff90c9e8d755fa639e8051a6eb3488')
 
 prepare() {
-  cd libtiff
+  cd tiff-${pkgver}
 
   # Rename 3.9.7 to 4.3.7, 3.6.x was the first release with the unintentional ABI change
   patch -Np1 -i ../libtiff4-soname.patch
@@ -32,7 +31,7 @@
 }
 
 build() {
-  cd libtiff
+  cd tiff-${pkgver}
 
   export CC='gcc -m32'
   export CXX='g++ -m32'
@@ -45,7 +44,7 @@
 }
 
 package() {
-  make DESTDIR="${pkgdir}" -C libtiff install
+  make DESTDIR="${pkgdir}" -C tiff-${pkgver} install
   rm -rf "${pkgdir}"/usr/{bin,include,lib32/libtiff{,xx}.{a,so},share}
   install -dm 755 "${pkgdir}"/usr/share/licenses
   ln -s libtiff4 "${pkgdir}"/usr/share/licenses/lib32-libtiff4
